src/Microsoft.Data/BaseColumn.DataFrameFunctions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data
+{
+    public abstract partial class BaseColumn
```

Comment:
I'm not sure I understand the name of this file - `BaseColumn.DataFrameFunctions`. Aren't all the functions in BaseColumn "data frame" functions?

Reply:
Yup, they are. I just wanted to organize these functions in a different file, similar to `BaseColumn.Computations` and `BaseColumn.BinaryOperations`. The idea was to add all new functions to this file. I didn't really pay much attention to the name here. I'm open to changing it. I was more interested in avoiding a big file like `DataFrame.cs` is now

Reply:
I’m not sure I like this reasoning.

1. It doesn’t solve the problem because as you add more functions to this file, it is going to fill up as well. 
2. People who are trying to get a grasp on the functionality need to jump back and forth between multiple files just to understand what this class does.